tests/lib/rules/no-test-import-export.js

```diff
@@ -24,6 +24,26 @@ ruleTester.run('no-test-file-importing', rule, {
         export function beforeEachSetup(){};
       `,
     },
+
+    // Exporting from files in tests/helpers is allowed.
+    {
+      filename: 'tests/helpers/setup-application-test.js',
+      code: 'export default function setupApplicationTest(){};',
+    },
+    {
+      filename: 'tests/helpers/index.js',
+      code: 'export function setupApplicationTest(){};',
+    },
+    {
+      filename: 'my-app-name/tests/helpers/setup-application-test.js',
+      code: 'export default function setupApplicationTest(){};',
+    },
+
+    // Importing anything from tests/helpers is allowed.
+    "import setupApplicationTest from 'tests/helpers/setup-application-test.js';",
```

Comment:
We are missing relative path test case, for example:

```
../../helpers/setup-application-test.js
```

--

I upgrade eslint-plugin-ember to v8.10.1 and my ESLint still complains about imports from helpers folder, because relative import paths are being used.

I can workaround by changing to absolute path (TypeScript app), like `myapp/tests/helpers/`.

It seems plugin should resolve real path on disk with `path.resolve` or something to determine truthiness in `isTestHelperFilename` method.

Reply:
This should fix it: https://github.com/ember-cli/eslint-plugin-ember/pull/911